src/commands/data/privatelink/access/add.ts

```diff
@@ -1,48 +1,45 @@
 import {flags} from '@heroku-cli/command'
-import {cli} from 'cli-ux'
-
+import {Args, ux} from '@oclif/core'
 import BaseCommand from '../../../../base'
 import fetcher from '../../../../lib/fetcher'
 
-export default class EndpointsAccessAdd extends BaseCommand {
+export default class Add extends BaseCommand {
   static description = 'add an allowed account to your privatelink endpoint'
```

Comment:
Can we clean up this help description to indicate you can add multiple accounts and they're AWS accounts? Like: `add one or more allowed AWS accounts to your privatelink endpoint`